### PR TITLE
Add CC for RFI regarding memory safe advocacy

### PR DESCRIPTION
**Pull request by markcmiller86**
EB member: @bartlettroscoe 
Author: @markcmiller86 

@rinkug this is a new CC

### CODE REVIEW

File: CuratedContent/NationalCyberDirectorRFI.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+> [py-cryptography](https://cryptography.io/en/latest/) and a number of other very significant packages in the ecosystem are doing that.
+ 
+> A final point to keep in mind is that most HPC applications are not (yet) *services*.
+> They don’t need the same security guarantees that a library that sits listening for input on a port from random users need.
+> That’s good and bad.
+> It is good because they are unlikely to be attacked in the same ways.
+> And, because supply chain security for HPC applications can set different priorities.
+> It is bad because it weakens the case for encouraging anyone to move to a memory safe language merely for the sake of security.
+
+Visiters of the [RFI](https://www.regulations.gov/document/ONCD-2023-0002-0001) page will note about 100 *comments* are available there.
```
Comment (tgamblin) [inline CuratedContent/NationalCyberDirectorRFI.md:59]:
```suggestion
Visitors to the [RFI](https://www.regulations.gov/document/ONCD-2023-0002-0001) page will note about 100 *comments* are available there.
```

File: CuratedContent/NationalCyberDirectorRFI.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+> It was C++11, 14, 17, and on.
+> We’re talking about *modern* C++.
+ 
+> It is fair to say that the GPU ecosystem in Rust and Go is seriously lacking in comparison.
+> In addition, there is the inertia of games and AI to overcome to get a viable GPU compute ecosystem built in those languages too.
+> Julia has some integration, being a very compute-focused language.
+> However, it remains to be seen if the community will build HPC codes in it...or, if it is as viable for performance portability as C++ has been.
+
+> Whether the successor to C++, in scientific computing, if there is one, is going to be Julia, Rust, or Go is an open question.
+> The answer depends strongly on what industry does and how well they can make fast, memory-safe alternatives to the algorithms and frameworks such as [RAJA](https://raja.readthedocs.io/en/develop/) or [Kokkos]9https://kokkos.github.io) DOE has built in C++.
```
[the remark this answers is not in the document]
Comment (markcmiller86) [reply CuratedContent/NationalCyberDirectorRFI.md]: Resolved in c83ccf54

**Comment by markcmiller86** [comment]
@rinkug or @bartlettroscoe this is not showing up on preview after I rebuild. I search for `cybersecurity` and don't find it. Any ideas?

**Comment by markcmiller86** [comment]
thanks @tgamblin for the review 💪🏻

**Comment by bartlettroscoe** [comment]
> @rinkug or @bartlettroscoe this is not showing up on preview after I rebuild. I search for `cybersecurity` and don't find it. Any ideas?

@markcmiller86, I see you rebuilt the preview.bssw.io site about an hour ago and it shows under "Files Successfully Prodcessed" it shows the file `CuratedContent/NationalCyberDirectorRFI.md`.   I can't explain why this article is not showing up.  It is not even showing up under your page:

* https://preview.bssw.io/items?author=mark-c-miller

I have no idea how to debug this.

**Comment by rinkug** [comment]
@bartlettroscoe: I toggled preview label and rebuilt and now its [**there**](https://preview.bssw.io/items/us-federal-government-effort-to-champion-adoption-of-memory-safe-languages)

**Comment by markcmiller86** [comment]
@bartlettroscoe thanks for sharing about Teuchos. I added a sentence and link about it at end.

@rinkug I think this is also ready now.

**Pull request merged**